dev/com.ibm.ws.concurrent.persistent_fat_multiple/publish/servers/com.ibm.ws.concurrent.persistent.fat.multiple/server.xml

```diff
@@ -0,0 +1,49 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+  <featureManager>
+    <feature>jndi-1.0</feature>
+    <!-- <feature>osgiConsole-1.0</feature> -->
+    <feature>persistentExecutor-1.0</feature>
+    <feature>servlet-3.1</feature>
+    <feature>componenttest-1.0</feature>
+  </featureManager>
+
+  <include location="../fatTestPorts.xml"/>
+
+  <persistentExecutor id="executor1" jndiName="concurrent/executor1" pollInterval="4s" retryLimit="1" pollSize="100"/>
+  <persistentExecutor id="executor2" jndiName="concurrent/executor2" taskStoreRef="defaultDatabaseStore" pollInterval="4s" retryLimit="1"/>
+
+  <databaseStore id="defaultDatabaseStore" keyGenerationStrategy="SEQUENCE"/>
+  
+  <library id="AnonymousJDBCLib">
+    <fileset dir="${shared.resource.dir}/jdbc" includes="${env.DB_DRIVER}"/>
+  </library>
+  
+  <!-- Default database, properties changed by FATRunner during database rotation. -->
+  <dataSource id="DefaultDataSource">
+    <jdbcDriver libraryRef="AnonymousJDBCLib"/>
+  	<properties.derby.embedded createDatabase="create" databaseName="memory:persistmultidb" user="userx" password="passx"/>
+  </dataSource>
+  
+  <javaPermission codebase="${shared.resource.dir}/jdbc/db2jcc.jar" className="java.security.AllPermission"/>
```

Comment:
instead of adding perms to all drivers, could we just have 1 `<javaPermission>` and re-use the `DB_DRIVER` variable to grant perms to the in-use driver like this?
```xml
<javaPermission codebase="${shared.resource.dir}/jdbc/${DB_DRIVER}" className="java.security.AllPermission"/>
```
this will help as drivers come/go or change names (e.g. ojdbc8_g.jar --> ojdbc9.jar)